### PR TITLE
set TargetPlatformMinVersion to 10.0.17134.0

### PR DESCRIPTION
**Pull request by huichen123**
WinUI needs to support RS4

### CODE REVIEW

File: dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/projection/Microsoft.ApplicationModel.Resources.Projection.csproj [hunk 1]
```diff
@@ -2,6 +2,7 @@
   
   <PropertyGroup>
     <TargetFrameworks>net5.0-windows10.0.18362</TargetFrameworks>
```
Comment (alwu-msft) [inline dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/projection/Microsoft.ApplicationModel.Resources.Projection.csproj:4]: Shouldn't this be `TargetFramework` (no `s`)?

**Comment by alwu-msft** [comment]
You can verify this on your end by opening the interop assembly in a disassembler (e.g. ILSpy) and checking that it has the following pair of asssembly-level attributes:
```csharp
[assembly: TargetPlatform("Windows10.0.18362.0")]
[assembly: SupportedOSPlatform("Windows10.0.17134.0")]
```

**Review by alwu-msft — APPROVED** [review]

**Comment by MikeHillberg** [comment]
Whoa, who/what looks at those attributes?

Do we need this for Preview3?

**Comment by alwu-msft** [comment]
It's a new code analyzer in .NET 5 (https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1416). We got tripped up by it ourselves, and setting the TargetPlatformMinVersion on the interop assembly is a change I'm making as part of our final ingestion.

**Pull request merged**